src/fragments/Slider.react.js

```diff
@@ -16,8 +17,20 @@ export default class Slider extends Component {
         this.DashSlider = props.tooltip
             ? createSliderWithTooltip(ReactSlider)
             : ReactSlider;
+        this.SyncedInput = Input;
```

Comment:
Probably don't need this indirection - `this.DashSlider` is abstracted because there are two different versions of the component depending on whether tooltips are requested (though it occurs to me, the way this is currently implemented it looks like there may be a bug if you add or remove `props.tooltip` dynamically?), but because `Input` only has one flavor you should be able to use it directly in the `render` method.

Reply:
Actually, looking at this a little more closely, what's the benefit of using our `Input` component instead of an HTML `input` tag directly? The `Input` is kind of confusing used this way, and if your used `input` you could attach a `ref` to it and avoid having to pass `event` around just to get the value out. (You can attach a `ref` to a react class as well, but I think that would make it even more confusing to get the value out, because this doesn't make its way back into state or props given the current usage AFAICT...)

Reply:
I agree, in hindsight it was confusing to re-use the `dcc.Input` component in this context. Instead, I have followed your suggestion and used a `ref` to a JSX-generated `input` HTML tag.